### PR TITLE
[profile] Implement v1 Profile Screen (screens/profile/ProfileScreen.tsx)

### PR DESCRIPTION
**Pull request by EthanAuyeung**
[//]: # "These comments are meant for your reference. They are invisible and don't need to be deleted!"

### Description
[//]: # "Required - Describe what's new in this PR in a few lines. A description and bullet points for specifics will suffice."
Implemented frontend changes to `ProfileScreen.tsx` as per v1 designs. 

## How to review
[//]: # "Required - Describe the order in which to review files and what to expect when testing locally. Is there anything specifically you want feedback on? Should this be reviewed commit by commit, or all at once? What are some user flows to test? What are some edge cases to look out for?"
Run npm start and play around w the app.

## Next steps
[//]: # "Optional - What's NOT in this PR, doesn't work yet, and/or still needs to be done. Note any temporary fixes in this PR that should be cleaned up later."
- Build Contact Us screen
- Implement email us functionality on the Contact screen

[//]: # "This tags the project leader as a default. Feel free to change, or add on anyone who you should be in on the conversation."
🥦 CC: @sauhardjain

### CODE REVIEW

File: src/screens/profile/styles.ts [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+export const HeadingContainer = styled.Text`
+  margin-top: 57px;
+  margin-bottom: 16px;
+  word-break: break-all;
+  width: 100%;
+`;
+export const ButtonInfoContainer = styled.TouchableOpacity`
+  margin-top: 4%;
+  margin-left: 0px;
+`;
```
Comment (sauhardjain) [inline src/screens/profile/styles.ts]: Unused style

File: src/screens/profile/styles.ts [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+  box-sizing: border-box;
+  height: 0px;
+  width: 110%;
+  border: 0.5px;
+`;
+export const ButtonBottomContainer = styled.TouchableOpacity`
+  margin-top: 3%;
+  margin-left: 0%;
+  width: 100%;
+`;
```
Comment (sauhardjain) [inline src/screens/profile/styles.ts]: Unused styles

File: src/screens/profile/styles.ts [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+  padding-right: 32px;
+  margin-bottom: 64px;
+`;
+export const BackButtonContainer = styled.View`
+  width: 100%;
+  justify-content: flex-start;
+  align-items: flex-start;
+  margin-left: 24px;
+  margin-top: 57px;
+`;
```
Comment (sauhardjain) [inline src/screens/profile/styles.ts]: Unused

File: src/screens/profile/styles.ts [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+export const PhoneHeadingContainer = styled.View`
+  margin-top: 30%;
+  margin-left: 15%;
+`;
+export const FormContainer = styled.View`
+  margin-horizontal: 24px;
+  display: flex;
+  align-items: center;
+`;
```
Comment (sauhardjain) [inline src/screens/profile/styles.ts]: Unused

**Review by sauhardjain — CHANGES_REQUESTED** [review]
Great work on this! There are a lot of unused styles from the screens we got rid of, make sure you remove these before opening a PR in the future.
